Tone/core/context/ToneAudioNode.ts

```diff
@@ -376,3 +377,16 @@ export function disconnect(
 		srcNode.disconnect();
 	}
 }
+
+/**
+ * connect the output of this node to the rest of the nodes in parallel.
+ * @example
+ * const player = new Tone.Player("https://tonejs.github.io/audio/drum-samples/conga-rhythm.mp3");
+ * const player1 = new Tone.Player("https://tonejs.github.io/audio/drum-samples/conga-rhythm.mp3");
+ * const filter = new Tone.Filter("G5").toDestination();
+ * // connect nodes to a common destination
+ * fanIn(filter, player, player1);
+ */
+export function fanIn(dstNode: InputNode, ...srcNodes: OutputNode[]): void {
```

Comment:
I think the API would make more sense if it was `fanIn(...srcNodes, dstNode)` so that it's consistent with the rest of the connect APIs which always have the sources first and the destination after

Reply:
That was my first thought also. The problem is that the rest parameter cannot come at the beginning of the function declaration – only the end.

How about this:

```typescript
export function fanIn(...nodes: any[]): void {
  const dstNode = nodes.pop();
  nodes.forEach(node => connect(node, dstNode));
}

fanIn(mySynth1, mySynth2, myReverb) // connects all to myReverb
```

The function declaration is less clear (and less strict), but it allows for parameters to include multiple `OutputNode`s and one `InputNode`.

Reply:
See latest commit.